themes/bootstrap3/templates/Recommend/FavoriteFacets.phtml

```diff
@@ -2,7 +2,7 @@
 <?php $sideFacetSet = $this->recommend->getFacetSet(); ?>
 
 <?php if (isset($sideFacetSet['tags']) && !empty($sideFacetSet['tags']['list'])): ?>
-  <h4 id="acc-menu-favs-header" class="tag"><?=$this->transEsc($sideFacetSet['tags']['label'])?></h4>
+  <h2 id="acc-menu-favs-header" class="tag"><?=$this->transEsc($sideFacetSet['tags']['label'])?></h2>
```

Comment:
This recommendation module is not documented in searches.ini so I'm not sure how to use / test it.

Reply:
This is used in the account area -- it displays the tag filters when looking at favorites and lists.

Reply:
Ok thanks found it.  In this case it should actually be `h3` for consistency with Your Lists and Your Account above, unless it was intentionally meant to be a subheading of Your Lists, which maybe?

Reply:
I guess you could argue either way regarding "Your Tags" being a subheading of "Your Lists," but I think keeping them parallel with `h3` makes sense.